Gemfile.lock

```diff
@@ -266,47 +266,48 @@ GEM
       minitest (>= 5.1)
       tzinfo (~> 2.0)
       zeitwerk (~> 2.3)
-    addressable (2.8.4)
+    addressable (2.8.6)
       public_suffix (>= 2.0.2, < 6.0)
-    airbrake (11.0.1)
+    airbrake (11.0.3)
       airbrake-ruby (~> 5.1)
-    airbrake-ruby (5.2.0)
+    airbrake-ruby (5.2.1)
       rbtree3 (~> 0.5)
     ansible (0.2.2)
     ar_multi_threaded_transactional_tests (0.5.0)
       activerecord (>= 4.2.0, < 6.2.0)
-    ast (2.4.1)
+    ast (2.4.2)
     attr_encrypted (3.1.0)
       encryptor (~> 3.0.0)
-    audited (5.4.2)
+    audited (5.4.3)
       activerecord (>= 5.0, < 7.2)
       request_store (~> 1.2)
-    autoprefixer-rails (9.4.8)
+    autoprefixer-rails (9.8.6.5)
       execjs
-    awesome_print (1.6.1)
-    aws-eventstream (1.2.0)
-    aws-partitions (1.329.0)
-    aws-sdk-core (3.100.0)
-      aws-eventstream (~> 1, >= 1.0.2)
-      aws-partitions (~> 1, >= 1.239.0)
+    awesome_print (1.9.2)
+    aws-eventstream (1.3.0)
+    aws-partitions (1.887.0)
+    aws-sdk-core (3.191.0)
+      aws-eventstream (~> 1, >= 1.3.0)
+      aws-partitions (~> 1, >= 1.651.0)
+      aws-sigv4 (~> 1.8)
+      jmespath (~> 1, >= 1.6.1)
+    aws-sdk-ecr (1.69.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-      jmespath (~> 1.0)
-    aws-sdk-ecr (1.3.0)
-      aws-sdk-core (~> 3)
-      aws-sigv4 (~> 1.0)
-    aws-sdk-kms (1.23.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-kms (1.77.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sigv4 (~> 1.1)
-    aws-sdk-s3 (1.45.0)
-      aws-sdk-core (~> 3, >= 3.58.0)
+    aws-sdk-s3 (1.143.0)
+      aws-sdk-core (~> 3, >= 3.191.0)
       aws-sdk-kms (~> 1)
-      aws-sigv4 (~> 1.1)
-    aws-sigv4 (1.6.1)
+      aws-sigv4 (~> 1.8)
+    aws-sigv4 (1.8.0)
       aws-eventstream (~> 1, >= 1.0.2)
-    base64 (0.1.1)
+    base64 (0.2.0)
```

Comment:
### [base64](https://github.com/ruby/base64)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.1.1 → 0.2.0

[_~~change-log~~, [source-code](https://github.com/ruby/base64), [gem-diff](https://my.diffend.io/gems/base64/0.1.1/0.2.0)_]





<details>
<summary>Commits</summary>

A change of **23** commits. See the full changes on [the compare page](https://github.com/ruby/base64/compare/v0.1.1...v0.2.0).

These are the first 10 commits:
- (888d885) [Add Ruby 3.1 to CI](https://github.com/ruby/base64/commit/888d88550d53958991063226d7aca5d76e49778d)
- (c0147a8) [Merge pull request #8 from petergoldstein/feature/add_ruby_3…](https://github.com/ruby/base64/commit/c0147a861c340865bf0eb6e0f94b1e383df9a922)
- (697a878) [Added dependabot](https://github.com/ruby/base64/commit/697a87822a025ae9ede82d44ca59b2c16f6f2171)
- (feb72b0) [Merge pull request #9 from ruby/add-dependabot](https://github.com/ruby/base64/commit/feb72b0ac993a95f935d2bb9bb76b18f033c89af)
- (872dd0a) [Bump actions/checkout from 2 to 3](https://github.com/ruby/base64/commit/872dd0a4e8603cdacaacfdbda95868339411a504)
- (5fd0018) [Merge pull request #10 from ruby/dependabot/github_actions/a…](https://github.com/ruby/base64/commit/5fd0018e4bae4a22578a723b951848fa6e2d8db3)
- (62db7e6) [Update Ruby versions on GHA with ci_versions/all.json](https://github.com/ruby/base64/commit/62db7e68de9eb8cc18e0b0c2f471513018546e23)
- (bf0530d) [Merge pull request #11 from ruby/ci-versions](https://github.com/ruby/base64/commit/bf0530d99643885a71041f31ce06192093aa7781)
- (21a636d) [Migrate set-output to $GITHUB_OUTPUT](https://github.com/ruby/base64/commit/21a636d1cad1608979d2c3cc49133f08b1d97812)
- (11288fc) [Merge pull request #12 from tanakakc/patch-1](https://github.com/ruby/base64/commit/11288fcdf9b043811ef6e69b908cbeaf95c71b7b)


</details>